ci/build/test-wheels.sh

```diff
@@ -87,20 +88,21 @@ if [[ "$platform" == "linux" ]]; then
 
   # Check that the other wheels are present.
   NUMBER_OF_WHEELS="$(find "$ROOT_DIR"/../../.whl/ -mindepth 1 -maxdepth 1 -name "*.whl" | wc -l)"
-  if [[ "$NUMBER_OF_WHEELS" != "5" ]]; then
+  if [[ "$NUMBER_OF_WHEELS" != "6" ]]; then
     echo "Wrong number of wheels found."
     ls -l "$ROOT_DIR/../.whl/"
     exit 2
   fi
 
 elif [[ "$platform" == "macosx" ]]; then
   MACPYTHON_PY_PREFIX=/Library/Frameworks/Python.framework/Versions
-  PY_WHEEL_VERSIONS=("36" "37" "38" "39" "310")
+  PY_WHEEL_VERSIONS=("36" "37" "38" "39" "310" "311")
```

Comment:
should I keep these?

Reply:
I guess we should revert it as well? Since py311 wheel for mac will not be built.

Or feel free to leave it if you are able to get macos wheel built successfully. I wasn't able to do so for CI. (You will need to unblock the mac pipeline here):

<img width="1182" alt="image" src="https://user-images.githubusercontent.com/11676094/210410629-5726ba34-8577-49d9-9d8b-c9c1dc66488b.png">

Reply:
I saw that option, but I don't have permissions to start the Mac build. Looks like the build stalled after a while.

I have been working on a Mac and it does build locally.

Reply:
Ah gotcha - I could do that. But yeah, I think it should be rather unlikely for mac build to regress.